ProteoformSuiteInternal/DeltaMassPeak.cs

```diff
@@ -37,6 +37,21 @@ public DeltaMassPeak(ProteoformRelation base_relation, List<ProteoformRelation>
             if (!Lollipop.opening_results && Lollipop.updated_theoretical) this.possiblePeakAssignments = nearestPTMs(this.peak_deltaM_average);
         }
 
+        //notch mass test
+        public DeltaMassPeak(bool notch, ProteoformRelation base_relation, List<ProteoformRelation> relations) : base(base_relation)
```

Comment:
Use the ConstructorsForTesting class in the Test project for these types of constructors.